### ISSUE TITLE
Support for Windows Server 1809 will be dropped, but there are no viable VMSS automatic OS upgrade images available

### ISSUE THREAD

**Issue by LeroyK** [microsoft/service-fabric#1145]
**Service Fabric Runtime Version:**
Future SF version

**Environment:**
Azure

**Description:** 

The [Service Fabric 7.2 Fifth Refresh Release Notes](https://github.com/microsoft/service-fabric/blob/master/release_notes/Service-Fabric-72CU5-ReleaseNotes.md) mentions that [support for Windows Server 1809 will be discontinued in future Service Fabric releases](https://github.com/microsoft/service-fabric/blame/7f30ccea5cbca3e6ecf3a55b3d1cf34d7c3bd143/release_notes/Service-Fabric-72CU5-ReleaseNotes.md#L36).

If support for 1809 is discontinued, there are no full server images available for VMSS automatic OS upgrade images. The only option would be the WindowsServer SKU 'Datacenter-Core-1903-with-Containers-smalldisk'.

I have also reported this at https://github.com/Azure/vm-scale-sets/issues/71, to make sure both teams are aware of this issue.

**OS(Windows/Linux):**
Windows

---
Assignees: /cc @microsoft/service-fabric-triage

**Comment by BaMi83** [comment]
yes i was just looking as well, and found out we are also using 2019 server with containers 1809 LTSC, looking further i noticed there is no new 2019 release which has LTSC. what is the best approach for this situation?

**Comment by peterpogorski** [comment]
Windows Server 2019 with containers is a LTSC release which does offer support for [VMSS automatic OS upgrades](https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-automatic-upgrade#:~:text=%20The%20recommended%20steps%20to%20recover%20VMs%20and,set%20model%20with%20any%20new%20tokens.%20More)

Have you been experiencing any issues when trying to use these automatic OS upgrades?

**Comment by BaMi83** [comment]
The releasenotes of CU5 tell us 1809 will be discontinued. The question is which server version we have to use? There is no new server version with ltsc available, only semi annual core versions.

> Windows Server 2019 with containers is a LTSC release which does offer support for [VMSS automatic OS upgrades](https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-automatic-upgrade#:~:text=%20The%20recommended%20steps%20to%20recover%20VMs%20and,set%20model%20with%20any%20new%20tokens.%20More)
> 
> Have you been experiencing any issues when trying to use these automatic OS upgrades?

**Comment by peterpogorski** [comment]
The LTSC version of 1809 is named Windows Server 2019 with containers in the VM OS SKU. The OS SKU named Windows Server 1809 was a SAC release which went out of support on 11/10/2020. See https://docs.microsoft.com/en-us/windows-server/get-started/windows-server-release-info for more info.

**Comment by BaMi83** [comment]
> The LTSC version of 1809 is named Windows Server 2019 with containers in the VM OS SKU. The OS SKU named Windows Server 1809 was a SAC release which went out of support on 11/10/2020. See https://docs.microsoft.com/en-us/windows-server/get-started/windows-server-release-info for more info.

Hi Peter Pogorski,

thanks for explaining, but will this say that windows server 2019 with containers 1809 (LTSC) isn't deprecated after CU5 and will it only be the SAC 1809 release which will be deprecated?

regards 

Bart

**Comment by craftyhouse** [comment]
Correct
This one is deprecated after CU5
Windows Server, version 1809 (Semi-Annual Channel) (Datacenter Core, Standard Core) | 1809 | 17763.107.1010129-1455 | 11/13/2018 | 11/10/2020 | Review note

This one is still supported
Windows Server 2019 (Long-Term Servicing Channel) (Datacenter, Essentials, Standard) | 1809 | 17763.107.1010129-1455 | 11/13/2018 | 01/09/2024 | 01/09/2029

Please let us know if we should clarify further in upcoming release notes.
